pkg/network/tracer/utils_linux.go

```diff
@@ -3,8 +3,6 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build linux_bpf
```

Comment:
should this file still be called `utils_linux`?

Reply:
Yes, this is still linux specific, just more generally for linux, not just linux_bpf.

Reply:
this would also include darwin right? I think there have been some conversations about supporting that in system-probe